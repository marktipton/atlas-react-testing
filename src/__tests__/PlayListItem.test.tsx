import { render } from "@testing-library/react";
import { expect, test } from "vitest";
import  PlayListItem  from "../components/PlayListItem";

test("volume control renders correctly w/", () => {
  const { container } = render(
  <PlayListItem
    name="Daily Routine"
    artist="Animal Collective"
    time="5:47"
    bgColor="bg-secondary-light"
    onClick={() => {}}
  />
  );
  expect(container).toMatchSnapshot();
});