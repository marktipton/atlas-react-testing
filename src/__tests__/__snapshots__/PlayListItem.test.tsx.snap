// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`volume control renders correctly w/ 1`] = `
<div>
  <div
    class="flex w-full items-center justify-between bg-secondary-light rounded-lg"
  >
    <div>
      <p
        class="text-primary font-medium"
      >
        Daily Routine
      </p>
      <p
        class="text-secondary font-medium"
      >
        Animal Collective
      </p>
    </div>
    <p
      class="text-secondary font-medium"
    >
      5:47
    </p>
  </div>
</div>
`;
